app/src/main/java/com/example/swugger/EditEventDialogFragment.java

```diff
@@ -0,0 +1,313 @@
+package com.example.swugger;
+
+import android.app.DatePickerDialog;
+import android.app.Dialog;
+import android.app.PendingIntent;
+import android.app.TimePickerDialog;
+import android.content.res.Resources;
+import android.os.Bundle;
+import android.support.v4.app.DialogFragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.DatePicker;
+import android.widget.EditText;
+import android.widget.ImageButton;
+import android.widget.ImageView;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+import android.widget.TimePicker;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Locale;
+
+public class EditEventDialogFragment extends DialogFragment implements DatePickerDialog.OnDateSetListener, TimePickerDialog.OnTimeSetListener, AddReminderDialogFragment.AddReminderDialogListener, RemindersAdapter.RemindersAdapterListener {
```

Comment:
This DialogFragment does so much stuff and I see repeated code with some other DialogFragments. Would it be possible for you to create a class that EditEventDialogFragment and the other DialogFragments would extend to preform basic set up?

Reply:
im not worried about refactoring code and making it as efficient and readable as i can right now, just making the basic functionality of the app, but later i will do stuff like this